README.md

```diff
@@ -80,6 +78,12 @@ If you're bootstrapping a Juju controller within a `multipass` VM you will not h
   - `juju dashboard` and take note of the ip:port
   - `ssh -fN -L *:17070:0:17070 ubuntu@10.223.241.32` Where the "10." ip is replaced with the ip from the output of the `juju dashboard` command.
 
-### Updating CRA
+### Writing tests
+
+...coming soon...
+
+### Developer notes
```

Comment:
Can we remove until we have something for here?

Reply:
The updating CRA segment is under this title :)